poetry/core/packages/package.py

```diff
@@ -152,7 +152,7 @@ def full_pretty_version(self) -> str:
 
         # if source reference is a sha1 hash -- truncate
         if len(self.source_reference) == 40:
-            return "{} {}".format(self._pretty_version, self.source_reference[0:7])
+            return f"{self._pretty_version} {self.source_reference[0:7]}"
 
         return "{} {}".format(
```

Comment:
It missing to apply the f-string in the last return of the method `full_pretty_version()`.